sdk/metricsadvisor/ai-metrics-advisor/samples/javascript/incidentsAndAlerts.js

```diff
@@ -30,6 +30,28 @@ async function main() {
   await listAlerts(client, alertConfigId);
   await listIncidentsForAlert(client, alertConfigId, alertId);
   await listAnomaliesForAlert(client, alertConfigId, alertId);
+  await listAnomalyDimensionValues(client, detectionConfigId);
```

Comment:
Move this above getRootCauses() for better grouping

Reply:
done